SharePointFramework/ProjectExtensions/src/projectSetup/index.ts

```diff
@@ -366,7 +366,7 @@ export default class ProjectSetup extends BaseApplicationCustomizer<IProjectSetu
 
       const [templates, extensions, listContentConfig] = await Promise.all([
         this._portal.getItems(
-          this.properties.templatesLibrary,
+          'Maloppsett', //Temp test value
```

Comment:
This doesn't look good ;) We should probably use input from properties?

Reply:
Woops. I have added the property to the extension, forgot to remove it form testing.